aws-lc-rs/src/aead.rs

```diff
@@ -396,6 +396,43 @@ impl<N: NonceSequence> SealingKey<N> {
             in_out,
         )
     }
+
+    /// Encrypts and signs (“seals”) data in place with extra plaintext.
+    ///
+    /// `aad` is the additional authenticated data (AAD), if any. This is
+    /// authenticated but not encrypted. The type `A` could be a byte slice
+    /// `&[u8]`, a byte array `[u8; N]` for some constant `N`, `Vec<u8>`, etc.
+    /// If there is no AAD then use `Aad::empty()`.
+    ///
+    /// The plaintext is given as the input value of `in_out` and `extra_in`. `seal_in_place()`
+    /// will overwrite the plaintext contained in `in_out` with the ciphertext. The `extra_in` will
+    /// be encrypted into the `extra_out_and_tag`, along with the tag.
+    /// The `extra_out_and_tag` length must be equal to the `extra_len` and `self.algorithm.tag_len()`.
+    ///
+    /// # Errors
+    /// `error::Unspecified` when `nonce_sequence` cannot be advanced.
+    ///
+    #[inline]
+    #[allow(clippy::needless_pass_by_value)]
+    pub fn seal_in_place_scatter<A>(
```

Comment:
Had an offline conversation with Justin, so will summarize it here so that we can get this closed out. Let's remove `SealingKey::seal_in_place_scatter`, and only have support for this feature on `LessSafeKey`. With that small modification we are open to accepting this feature.

Reply:
Works for me!